server.js

```diff
@@ -52,15 +69,16 @@ app.get('/location', (req, res) => {
 //   dataFromLocationJSON[0].lon
 // );
 
-
-
+//refactor code to reference WEATHER_API_KEY from .env file like the note I wrote on line 34
+//array.prototype.map on weatherData arr
 app.get('/weather', (req, res) => {
   const weatherData = require('./data/weather.json');
   const arr2 = [];
   weatherData.data.forEach(weatherObject => {
     const newWeatherObj = new Weather(weatherObject.weather.description, weatherObject.valid_date);
```

Comment:
We can write this a little cleaner by accessing the data in the constructor function like we talked about in the code review.